keras_hub/src/utils/transformers/export/hf_exporter.py

```diff
@@ -18,46 +21,38 @@
     # Add future models here, e.g., "LlamaBackbone": get_llama_weights_map,
 }
 
+MODEL_TOKENIZER_CONFIGS = {
+    "GemmaTokenizer": get_gemma_tokenizer_config,
+    # Add for future models, e.g., "LlamaTokenizer": get_llama_tokenizer_config,
+}
 
-def export_to_safetensors(keras_model, path):
-    """Converts a Keras model to Hugging Face safetensor format.
-
-    It does the following:
-    - Extracts and maps weights from the Keras backbone to safetensors.
-    - Saves the configuration as 'config.json'.
-    - Saves weights in 'model.safetensors'.
-    - Saves tokenizer assets.
 
+def export_backbone(backbone, path):
```

Comment:
One question I have is:

In this case, just having `export_backbone` works because the head is a transpose of the input embedding. But we have models for which the head is not tied. We need to take care of those too.